tests/parsers/test_sql_directory_parser.py

```diff
@@ -98,3 +98,10 @@ def test_parse_frontmatter(self):
             new_customers_table.operator.sql
             == "SELECT * FROM {customers_table} WHERE member_since > DATEADD(day, -7, '{{ execution_date }}')"
         )
+
+    def test_parse_creates_xcom(self):
+        with self.dag:
+            rendered_tasks = aql.render(dir_path + "/single_task_dag")
+        from tests.operators import utils as test_utils
```

Comment:
What's the benefit in having the import statement from within the test function?

Reply:
None I can move that